src/pages/stacks-blockchain/sending-tokens.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Sending tokens
+description: Learn how to transfer tokens
```

Comment:
This document is very NodeJS specific. We should call that out in the Introduction?

Separately, I wonder if it makes sense to either add some language agnostic description and/or link out to where people can get more information (e.g. the relevant SIPs)?

Reply:
Good points, will call out both!